nx-X11/programs/Xserver/os/Imakefile

```diff
@@ -121,16 +124,51 @@ TMEMCMP_OBJS = timingsafe_memcmp.o
 #endif
 
 BOOTSTRAPCFLAGS = 
-           SRCS = WaitFor.c access.c connection.c io.c $(COLOR_SRCS) \
-                  osinit.c utils.c log.c auth.c mitauth.c secauth.c \
-                  $(XDMAUTHSRCS) $(RPCSRCS) xdmcp.c OtherSources \
-                  xstrans.c $(SNPRINTF_SRCS) $(STRLCAT_SRCS) \
-		  $(REALLOCARRAY_SRCS) xprintf.c $(TMEMCMP_SRCS)
-           OBJS = WaitFor.o access.o connection.o io.o $(COLOR_OBJS) \
-                  osinit.o utils.o log.o auth.o mitauth.o secauth.o \
-                  $(XDMAUTHOBJS) $(RPCOBJS) xdmcp.o OtherObjects \
-                  xstrans.o $(SNPRINTF_OBJS) $(STRLCAT_OBJS) \
-		  $(REALLOCARRAY_OBJS) xprintf.o $(TMEMCMP_OBJS)
+           SRCS = WaitFor.c		\
+                  access.c		\
+                  connection.c		\
+                  io.c			\
+                  $(COLOR_SRCS)		\
```

Comment:
> You've broken the escape character lineup with tabsize = 8 here and on several occasions below that line.
>
> I guess you don't see it because you set your editor to a different tabsize, just mentioning it.
> 
> Also, the pkg-config line way down is too long even for tabsize = 8, so while you're at it, make sure the escape chars line up in that hunk?

@Ionic:I use tab size 8 here and the breakage only appears in the diff output (because of the extra "+" at the beginning of the diff line. In the file itself, the tabbing is ok.